complete-config.yaml

```diff
@@ -0,0 +1,145 @@
+      # If `null`, this will be auto-generated.
+      certificateChainFile: null
+      # Filepath to the `privkey.pem` of the certificate of your domain.
+      # If `null`, this will be auto-generated.
+      keyFile: null
+      # Network interface that the node listens on. `*` is a safe default.
+      interface: '*'
+
+      # You.
+      hostaddress:
+        # This should be your public IP or domain name.
+        hostname: localhost
+        # The port you'd like to run the Node on. 443 is a safe default.
+        port: 0
+
+    # When `true`, only communicates with peers specified in the `peers` list.
+    # Intended for "focused" catch up.
+    private: false
+
+    # The maximum number of peers to connect to at any one time.
+    maxSessionCount: 10
+
+    # The number of seconds to spend connected to a single peer.
+    # Don't alter this unless you know what you're doing.
+    sessionTimeout: 240
+
+    # Initial peers to connect to in order to join the network for the first time.
+    # These will share more peers and block data to your Node.
+    peers:
+    - address:
+        hostname: us-w1.chainweb.com
+        port: 443
+      id: null
+    - address:
+        hostname: us-e1.chainweb.com
+        port: 443
+      id: null
+    - address:
+        hostname: jp1.chainweb.com
+        port: 443
+      id: null
+    - address:
+        hostname: fr1.chainweb.com
+        port: 443
+      id: null
+
+  # Related to Cut DB management and synchronization between peers.
+  cuts:
+    # If `true`, delete block data that is not part of the main chain.
+    pruneChainDatabase: true
+    # The amount of microseconds to wait before abandoning an attempt to fully
+    # sync a Cut from some peer.
+    # Don't alter this unless you know what you're doing.
+    fetchTimeout: 3000000
+    # Force your node back to a certain Cut height, and have it resync from there.
+    initialCutHeightLimit: null
+
+  # If `true`, allow your node to feed data to a block explorer.
+  headerStream: false
+
+# --- Logging --- #
+
+logging:
+  # All structural (JSON, etc.) logs.
+  telemetryBackend:
+    enabled: true
+    configuration:
+      handle: stdout
+      color: auto
+      # `text` or `json`
+      format: text
+
+  # Simple text logs.
+  backend:
+    handle: stdout
+    color: auto
+    # `text` or `json`
+    format: text
+
+  logger:
+    log_level: info
+
+  filter:
+    rules:
+      - key: component
+        value: cut-monitor
+        level: info
+      - key: component
+        value: pact-tx-replay
+        level: info
+      - key: component
+        value: connection-manager
+        level: info
+      - key: component
+        value: miner
+        level: info
+      - key: component
+        value: local-handler
+        level: info
+    default: error
+
+# --- Miscellaneous Settings --- #
+
+# Location of your database files. Leaving this as `null` tells Chainweb to use
+# its default locations.
+databaseDirectory: null
```

Comment:
Should we take this out or suggest that people not use this until the DB location bug is fixed?

Reply:
The fix only applies to the default location, I think. @larskuhtz ?

Reply:
The default location puts the DBs in the right place.  I believe the bug happens when people override it.

Reply:
I'll hide it for now.